packages/shared/src/utils/route/route.ts

```diff
@@ -1,8 +1,8 @@
 // Checks if pathname matches route. (Works even with query string /?)
-import React from 'react';
+
 // TODO: Add test cases for this
-type TRoute = {
-    component?: JSX.Element | null;
+export type TRoute = {
+    component?: React.ElementType | null;
```

Comment:
Would it be better to use just React Node here?

Reply:
React node takes a string as well. So I avoided it